package.json

```diff
@@ -32,12 +33,28 @@
     "url": "http://github.com/robertknight/webpack-bundle-size-analyzer.git"
   },
   "devDependencies": {
-    "@types/chai": "^3.4.34",
     "@types/commander": "^2.3.31",
-    "@types/mocha": "^2.2.36",
+    "@types/jest": "^19.2.2",
     "@types/node": "^6.0.58",
-    "chai": "^3.5.0",
-    "mocha": "^3.2.0",
+    "jest": "^19.0.2",
```

Comment:
I've been happy with `jest`, mainly for the snapshotting feature, and for the fact that I only have to include 1 testing library (previously I'd have things like mocha, chai, sinon, nyc, mockery, etc).